doc/development_process/review_process.rst

```diff
@@ -0,0 +1,215 @@
+  not be merged without your approval.
+
+
+Pull Requests should have at least 2 approvals before they are merged
+======================================================================
+
+A pull-request shall be merged only with two positive reviews (approval). Beside
+the person merging the pull-request (merging != approval), two additional
+approvals are required to be able to merge a pull request. The person merging
+the request can merge without approving or approve and merge to get to the 2
+approvals required.
+
+Reviewers should keep track of pull requests they have provided feedback to
+===========================================================================
+
+If a reviewer has requested changes in a pull request, he or she should monitor
+the state of the pull request and/or respond to mention requests to see if his
+feedback has been addressed. Failing to do so, negative reviews shall be
+dismissed by the assignee or an owner of the repository. Reviews will be
+dismissed following the criteria below:
+
+- The feedback or concerns were visibly addressed by the author
+- The reviewer did not revisit the pull request after 1 week and multiple pings
+  by the author
+- The review is unrelated to the code change or asking for unjustified
+  structural changes such as:
+
+  - Split the PR
+  - Split the commits
+  - Can you fix this unrelated code that happens to appear in the diff
+  - Can you fix unrelated issues
+  - Etc.
+
+Closing Stale Issues and Pull Requests
+=======================================
+
+- The Pull requests and issues sections on Github are NOT discussion forums.
+  They are items that we need to execute and drive to closure.
+  Use the mailing lists for discussions.
+- In case of both issues and pull-requests the original poster needs to respond
+  to questions and provide clarifications regarding the issue or the change.
+  Failing to do so, an issue or a pull request will be closed automatically
+  after 6 months.
+
+Continuous Integration
+***********************
+
+All changes submitted to GitHub are subject to sanity tests that are run on
+emulated platforms and architectures to identify breakage and regressions that
+can be immediately identified. Sanity testing additionally performs build tests
+of a representative number of boards and platforms (subject to hardware
+availability in the CI infrastructure). Documentation changes are also verified
+through review and build testing to verify doc generation will be successful.
+
+Any failures found during the CI test run will result in a negative review
+assigned automatically by the CI system.
+Developers are expected to fix issues and rework their patches and submit again.
+
+The CI infrastructure currently runs the following tests:
+
+- Run ''checkpatch'' for code style issues (can vote -1 on errors)
+- Gitlint: Git commit style based on project requirements
+- License Check: Check for conflicting licenses
+- Run ''sanitycheck'' script
```

Comment:
Also BLE tests in BabbleSim

Reply:
not going into the details here. The above is part of sanity testing